x-pack/plugins/ml/public/settings/calendars/edit/events_table/events_table.js

```diff
@@ -34,6 +34,7 @@ function DeleteButton({ onClick }) {
 }
 
 export function EventsTable({
+  canCreateCalendar,
```

Comment:
This needs to be passed `canDeleteCalendar` too, so that the Delete buttons in each row can be disabled as appropriate. Currently the links are enabled for a user without `canDeleteCalendar`:

![image](https://user-images.githubusercontent.com/7405507/50151158-f8537300-02b7-11e9-98cb-4373641242d9.png)

Reply:
Okay, adding it in! 👍